packages/editor/src/client/panel/userEditPanel.tsx

```diff
@@ -49,10 +48,10 @@ export function UserEditPanel({id, onClose, onSave}: UserEditPanelProps) {
   const [active, setActive] = useState(true)
   const [roles, setRoles] = useState<FullUserRoleFragment[]>([])
   const [userRoles, setUserRoles] = useState<FullUserRoleFragment[]>([])
-  const [subscription, setUserSubscription] = useState<FullUserSubscriptionFragment>()
+  // const [subscription, setUserSubscription] = useState<FullUserSubscriptionFragment>()
```

Comment:
Do this commented out code needs to stay there ?

Reply:
Wondering the same :)